src/main/scala/org/ergoplatform/wallet/protocol/context/ErgoContext.scala

```diff
@@ -0,0 +1,36 @@
+package org.ergoplatform.wallet.protocol.context
+
+import org.ergoplatform.wallet.protocol.Constants
+import org.ergoplatform.{ErgoLikeContext, ErgoLikeTransactionTemplate, UnsignedInput}
+import scorex.crypto.authds.ADDigest
+import sigmastate.interpreter.ContextExtension
+import sigmastate.{AvlTreeData, AvlTreeFlags}
+
+class ErgoContext(val stateContext: ErgoLikeStateContext,
```

Comment:
Is it possible to simplify this class to reduce the number of classes with context? Do we really need to distinct `ErgoLikeStateContext` and `TransactionContext`? Or maybe we don't even need `ErgoContext` class here and can use ErgoLikeContext from sigma?